crates/trigger/src/cli.rs

```diff
@@ -255,6 +255,12 @@ fn warn_if_wasm_build_slothful() -> sloth::SlothGuard {
     sloth::warn_if_slothful(SLOTH_WARNING_DELAY_MILLIS, format!("{message}\n"))
 }
 
+fn help_heading<E: TriggerExecutor>() -> Option<&'static str> {
+    let heading = format!("{} TRIGGER OPTIONS", E::TRIGGER_TYPE.to_uppercase());
+    let as_str = Box::new(heading).leak();
```

Comment:
Using `Box::leak` is fine here IMO, but I think perhaps using a static and `OnceCell` is another, arguably less hacky, solution.

Reply:
Agreed; `leak` is perfectly fine for this purpose.